controllers/comments-controller.js

```diff
@@ -25,4 +26,13 @@ function postComment(req, res, next) {
     .catch(next);
 }
 
-module.exports = { getComments, postComment };
+function deleteComment(req, res, next) {
+  const { comment_id } = req.params;
+  deleteCommentData(comment_id)
+    .then((body) => {
+      res.status(204).send({ body });
+    })
+    .catch(next);
```

Comment:
It's typically not the case that you need to respond with the deleted object for a DELETE endpoint, so you could do either:
- `res.status(204).send();`
-  or `res.sendStatus(204);`

To clean things up slightly